src/routes/customer-resource/customer-resource-show.js

```diff
@@ -1,53 +1,160 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import fetch from 'isomorphic-fetch';
+import { connect } from 'react-redux';
+import { handleActions } from 'redux-actions';
+import  { Observable }  from 'rxjs/Observable';
+import { bindActionCreators } from 'redux'
+
+import 'rxjs/add/observable/from';
+import 'rxjs/add/operator/catch';
+import 'rxjs/add/operator/switchMap';
```

Comment:
Are we using switchMap?

Reply:
We should. That way concurrent requests only result in a single `LOADED` action

Reply:
so i read up on `switchMap` and vaguely get it, but far from completely understand the difference.  i converted the `mergeMap`s to `switchMap`s and everything still seems sturdy.